torch/testing/_internal/common_distributed.py

```diff
@@ -957,21 +957,30 @@ def world_size(self) -> int:
 
 
 class SaveForwardInputsModule(nn.Module):
-    def __init__(self, forward_inputs: Dict[nn.Module, torch.Tensor]) -> None:
+    def __init__(
+        self,
+        forward_inputs: Dict[nn.Module, torch.Tensor],
+        convert_inputs: bool = False,
```

Comment:
nit: Let us make this non-defaulted since we always pass it explicitly in the unit tests?